### PR TITLE
Add the possibility to launch a state estimation on a network

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/test/java/org/gridsuite/study/server/NetworkModificationTest.java [hunk 1]
```diff
@@ -126,6 +126,9 @@ public class NetworkModificationTest {
 
     private static final String VOLTAGE_INIT_STATUS_JSON = "{\"status\":\"COMPLETED\"}";
 
+    private static final String STATE_ESTIMATION_RESULT_UUID = "d3a85e9b-9894-4255-8ec7-07ea965d24eb";
+    private static final String STATE_ESTIMATION_STATUS_JSON = "\"COMPLETED\"";
```
Comment (anistouri) [inline src/test/java/org/gridsuite/study/server/NetworkModificationTest.java:130]: Why is it not consistent with the other calculations?
{\"status\":\"COMPLETED\"}

Comment (dbraquart) [reply src/test/java/org/gridsuite/study/server/NetworkModificationTest.java:130]: "COMPLETED" is what is returned by the estim computation server !
I guess we should change all the others to be consistant ;-)

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Review by anistouri — APPROVED** [review]
Code OK

**Pull request merged**